orders/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db       import models
+
+from users.models    import User
+from products.models import Product, Option
+
+class Cart(models.Model):
+    user     = models.ForeignKey(User, on_delete=models.CASCADE)
+    product  = models.ForeignKey(Product, on_delete=models.CASCADE)
+    quantity = models.IntegerField()
+    option   = models.ForeignKey(Option, on_delete=models.SET_NULL, null=True)
+
+    class Meta:
+        db_table = 'carts'
+
+class OrderStatus(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'orderstatus'
```

Comment:
```suggestion
        db_table = 'order_status'
```
두 단어로 이루어진 테이블명은 언더바로 구분해주세요. 아래 ItemStatus, OrderItems 모두 동일한 경우이니 적용해주세요.

Reply:
넵 수정했습니다!